lib/ricohapi/mstorage/client.rb

```diff
@@ -9,6 +9,8 @@ class Client
       class Error < StandardError; end
       class Unauthorized < Error; end
 
+      User_meta_regex = /^user\.([A-Za-z0-9_\-]{1,256})$/
```

Comment:
I think you mean that defining constant's name with only uppercase is usual practice in Ruby.
So I'll fix it.